controller/token.go

```diff
@@ -208,15 +208,20 @@ func (c *TokenController) Retrieve(ctx *app.RetrieveTokenContext) error {
 
 // Status checks if the stored external provider token is available.
 func (c *TokenController) Status(ctx *app.StatusTokenContext) error {
-	_, errorResponse, err := c.retrieveToken(ctx, ctx.For, ctx.RequestData, ctx.ForcePull)
+	appToken, errorResponse, err := c.retrieveToken(ctx, ctx.For, ctx.RequestData, ctx.ForcePull)
 	if errorResponse != nil {
 		ctx.ResponseData.Header().Set("Access-Control-Expose-Headers", "WWW-Authenticate")
 		ctx.ResponseData.Header().Set("WWW-Authenticate", *errorResponse)
 	}
 	if err != nil {
 		return jsonapi.JSONErrorResponse(ctx, err)
 	}
-	return ctx.OK([]byte{})
+
+	tokeStatus := &app.ExternalTokenStatus{
```

Comment:
minor typo.

Reply:
Done in https://github.com/fabric8-services/fabric8-auth/pull/335/commits/fbf9b742b9b1f8a8500cfdd8416e6fa7669d5641